src/Builder/DocBlockDriverFactory.php

```diff
@@ -26,7 +26,7 @@ public function __construct(DriverFactoryInterface $driverFactoryToDecorate, ?Pa
         $this->typeParser = $typeParser;
     }
 
-    public function createDriver(array $metadataDirs, Reader $annotationReader): DriverInterface
+    public function createDriver(array $metadataDirs, ?Reader $annotationReader = null): DriverInterface
```

Comment:
This one line is why there is a hard B/C break; widening the annotation reader argument to allow null causes compile errors in implementing classes.